docs/source/getting_started.rst

```diff
@@ -112,45 +118,57 @@ acme1), you will get an SSL error unless you disable the SSL verification:
         conda config --set ssl_verify false
         binstar config --set ssl_verify False
 
-4. Set the following:
-
-    ::
-
-        conda config --add channels conda-forge
-        conda config --set channel_priority strict
-
-5. Once conda is properly working, you can install the **(a) Latest Stable Release** or
+4. Once conda and mamba are properly working, you can install the **(a) Latest Stable Release** or
 create a **(b) Development Environment**.
 
 (a) Latest Stable Release
 =========================
 
-Installation using conda
+Installation using mamba
 ------------------------
 
 First, make sure that you're using ``bash``. ::
 
-   $ bash
+   bash
 
-You must have Anaconda installed as well.
+You must have a conda base enviornment installed as well.
 See :ref:`"Installation in a Conda Environment" <conda_environment>` section above for
 installing conda.
-Create a new Anaconda environment with ``zppy`` installed and activate it: ::
 
-   $ conda create -n zppy_env -c e3sm -c conda-forge zppy
-   $ source activate zppy_env
+Create a new conda environment with ``zppy`` installed and activate it: ::
 
-Or you can install ``zppy`` in an existing environment. ::
+   mamba create -n zppy_env zppy
```

Comment:
@xylar Thanks for putting this together. I'm following along with the instructions but at this command I get `bash: mamba: command not found`.

Reply:
Did you install Mambaforge?  If not, did you got to the `Troubleshooting` section below?